crates/web-sys/webidls/enabled/RTCRtpReceiver.webidl

```diff
@@ -17,6 +17,7 @@ interface RTCRtpReceiver {
   sequence<RTCRtpContributingSource>    getContributingSources();
   [Pref="media.peerconnection.rtpsourcesapi.enabled"]
   sequence<RTCRtpSynchronizationSource> getSynchronizationSources();
+  attribute DOMHighResTimeStamp?        jitterBufferTarget;
```

Comment:
This should be marked with `[SetterThrows]`, which I've went ahead and implemented in #3998.
We can go ahead and merge this PR anytime though, whichever PR gets merged first can rebase on the other.

Reply:
Fixed.